deployment/build-s3-dist.sh

```diff
@@ -273,9 +273,33 @@ echo "--------------------------------------------------------------------------
 # Note: do not install using global (-g) option. This makes build-s3-dist.sh difficult
 # for customers and developers to use, as it globally changes their environment.
 
-STAGE=testEnv
+STAGE=dev
 echo Stage set to $STAGE
 
+# Create sample dev.yaml file with a random user pool domain
+# Remove dev.yaml if it exists, we'll start with a known config that works
+do_cmd rm -f $source_dir/swb-reference/src/config/dev.yaml
+
+cognitoDomainRandomString=$(xxd -l 5 -c 5 -p < /dev/urandom)
+regionShortNameRandomString=$(xxd -l 2 -c 2 -p < /dev/urandom)
+
+echo "
+# Stage Name
+stage: dev
+awsRegion: ''           # Keep this empty so that template picks it up automatically at deploy time
+awsRegionShortName: $regionShortNameRandomString      # We do this since region is unknown at synth time, but we still need shortname for resource naming before deploying
+rootUserEmailParamStorePath: '/swb/dev/rootUser/email'  # This is a sample path and will need to be updated before running post deployment
```

Comment:
Same as above, though will this fail for single click if someone doesn't set this up ahead of time?